tasks/run_cd4pe_job.rb

```diff
@@ -200,9 +200,23 @@ def initialize(working_dir:, job_token:, web_ui_endpoint:, job_owner:, job_insta
         file.write(ca_cert)
       end
     end
+
+    set_home_env_var
+    set_repo_dir_env_var
     
   end
 
+  def set_home_env_var
+    if (@windows_job)
+      ENV['HOME'] = run_system_cmd("powershell (Resolve-Path ~")[:message]
+    else
+      ENV['HOME'] = run_system_cmd("(echo ~)")[:message]
```

Comment:
We should use `$HOME` instead, so like `echo $HOME` and `(Resolve-Path $HOME)` (valid powershell?).

The `~` is a convenience provided by your shell to expand to `$HOME` so you don't have to type that out yourself.

Reply:
But then that makes me think something else is going on....

🤔